lib/promiscuous_black_hole/record.rb

```diff
@@ -34,18 +31,23 @@ def existing_record
 
     def update
       Promiscuous.debug "Updating record: [ #{attributes} ]"
-      criteria.update(formatted_attributes)
+      criteria.update(attributes)
     end
 
     def create
       Promiscuous.debug "Creating record: #{attributes.values}"
-      DB[table_name].insert(formatted_attributes)
+      DB[table_name].insert(attributes)
     end
 
-    def formatted_attributes
-      attrs = {}
-      attributes.each { |k, v| attrs[k] = sql_representation_for(v) }
-      attrs
+    def attributes
+      return @attributes if @attributes
+
+      @attributes = @raw_attributes.dup
+      @attributes.each do |k, v|
+        if v.is_a?(Hash)
+          @attributes[k] = v.to_json
```

Comment:
`to_json` here is using MultiJson right? Just want to make sure we're using the fastest possible JSON encoder